src/Containers/KitchenCard/constants.ts

```diff
@@ -1,19 +1,17 @@
-export interface Choice{
+export interface ModifierChoiceInput {
     name: string;
     choice_type: string;
 }
-
-export interface ItemModifier {
+export interface OrderModifier {
     name?: string;
-    choices: Choice[];
+    choices: ModifierChoiceInput[];
 }
-
-export interface Item {
+export interface OrderItem {
     name: string;
-    modifiers: ItemModifier[];
+    modifiers: OrderModifier[];
 }
-export enum ModifierChoiceTypeEnum {
-    ADD = 'ADD',
-    NO = 'NO',
-    DEFAULT = 'DEFAULT'
+export declare enum ModifierChoiceTypeEnum {
```

Comment:
Can't you import from TS SDK? @DenisMela can we move this

Reply:
I mean it shouldn't becuase this shouldn't touch data but maybe we should separate TS SDK with an @ types lol I learned something new today. This is ok then